internal-packages/next-dev/src/index.ts

```diff
@@ -73,7 +73,7 @@ async function instantiateMiniflare(
 	};
 
 	const devBindingsDurableObjectOptions = Object.fromEntries(
-		Object.entries(options.bindings).filter(
+		Object.entries(options.bindings ?? {}).filter(
```

Comment:
If it is possible to have no bindings in the options, then we should update the type of `DevBindingsOptions`.

Reply:
as we discussed I added assertions to make sure that the `options` structure is what we expect 🙂👍